src/test/shell/data/i18n/fr.json

```diff
@@ -1112,15 +1113,22 @@
       "INSTANCE_NO" : "Number of instances",
       "PER_PAGE" : "Bulks per page",
       "DEPLOYED" : "Deployed application",
-      "REMOVE" : "Remove"
+      "REMOVE" : "Remove",
+      "DOMAIN_ID" : "Domain id",
+      "DOMAIN_NAME" : "Domain name",
+      "DOMAIN_CODENAME" : "Domain codename",
+      "USER_ID" : "User id",
+      "EMAIL" : "Email",
+      "USER_NAME" : "User name"
     },
     "DOMAIN" : {
       "INFO" : "Additional information will be added later",
       "SUMMARY" : "Table present summary of your bulk domains and domain admin deployments",
       "HEADER" : "Bulk domain deployments",
       "DEPLOYMENT_IN_PROGRESS": "Bulk deployment in progress...",
       "NAVIGATION" : "New bulk domain deployment",
-      "DEPLOYMENTS" : "Domain deployments"
+      "DEPLOYMENTS" : "Domain deployments",
+      "UPLOAD" : "Select csv file with domain and administrator specification for bulk deployment"
```

Comment:
Select csv file with domain specification and domain administrator user information for bulk deployment